zipkin-collector/kafka/src/test/java/zipkin2/collector/kafka/KafkaCollectorRule.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2015-2020 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin2.collector.kafka;
+
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.AdminClientConfig;
+import org.apache.kafka.clients.admin.NewTopic;
+import org.junit.AssumptionViolatedException;
+import org.junit.ClassRule;
+import org.junit.rules.ExternalResource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.FixedHostPortGenericContainer;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+
+import java.util.Collections;
+import java.util.Properties;
+import java.util.concurrent.ExecutionException;
+
+/** This should be used as a {@link ClassRule} as it takes a very long time to start-up. */
+class KafkaCollectorRule extends ExternalResource {
+  static final Logger LOGGER = LoggerFactory.getLogger(KafkaCollectorRule.class);
+  static final String IMAGE = "openzipkin/zipkin-kafka";
```

Comment:
in others, we were specifying version tag. latest might be fine even if not necessarily repeatable... it would be less maintenance (and less thrashy than "master" tag) wdyt cc @anuraaga

Reply:
Would recommend a tag for reproducible build, yeah

Reply:
Adding `latest`